calm_adapter/window_generator/window_generator.py

```diff
@@ -0,0 +1,71 @@
+#!/bin/env python3
+
+from datetime import date, timedelta
+import click
+import json
+import boto3
+from tqdm import tqdm
```

Comment:
minor: Python style is to group imports into three chunks: standard library, third-party, our own code. See https://www.python.org/dev/peps/pep-0008/

```python
from datetime import date, timedelta
import json

import boto3
import click
from tqdm import tqdm
```